api/src/app/modules/project/project.service.ts

```diff
@@ -307,6 +309,7 @@ export class ProjectService extends DataService<Project, Repository<Project>, Pr
 
     const options = {relations: []};
     options.relations.push('submissions'); // add this extra relation for later use.
+		options.relations.push('publicNotices');
```

Comment:
why are we doing this?

Reply:
Interesting... not sure why on the pr the line does not align but on vscode it is....

During the workflow change request (when 'PUBLISH' is pushed/requested), it needs to add one more relation ('publicNotices') to load from TypeORM (like previously the 'submission' relation) so it can validate the public_notice.post_date later. If this is not added, relation will be null (even though database exists the relation).
We could load is separately later during validation but loading it initially should make more sense.
![image](https://github.com/bcgov/nr-fom/assets/81595625/93f731af-6be2-40ff-bee7-9fd9c93c39b1)

Reply:
This is fine.